loguru.hpp

```diff
@@ -90,6 +90,8 @@ Website: www.ilikebigbits.com
 	#pragma GCC system_header
 #endif
 
+#include <stddef.h> // For standard definitions i.e. size_t type definition
```

Comment:
Sorry, no. One of the design goals of Loguru is to keep `loguru.hpp` as light as possible, which mean having no `#include`s in it *at all*.

Reply:
Duly noted. I have instead casted the length parameter when used in the snprintf calls.